libs/vscode/webview/src/lib/get-task-execution-schema.ts

```diff
@@ -190,20 +192,20 @@ function getConfigValuesFromContextMenuUri(
     const appsDir = nxConfig.workspaceLayout?.appsDir ?? 'apps';
     const libsDir = nxConfig.workspaceLayout?.libsDir ?? 'libs';
     if (
-      (appsDir && schematic.name === 'application') ||
-      schematic.name === 'app'
+      (appsDir && generator.name === 'application') ||
```

Comment:
This is an example of a place where we do check if application || app instead of just the one.

Theoretically this shouldn't be necessary, "well behaved" plugins should stick to application and have app as an alias, but nx-dotnet doesn't so I am not sure who else does.

Reply:
I was thinking about changing this to use the new `GeneratorType`, but I didn't want to break existing expectations.

Reply:
Yeah, this ones not a recommended change, just a shout out to the comment on read-generator-collections